### DISCUSSION TITLE
what's the font is in example?

### DISCUSSION THREAD

**Discussion by caiyanhu** [nvim-treesitter/nvim-treesitter#3541] [General]
https://raw.githubusercontent.com/nvim-treesitter/nvim-treesitter/master/assets/example-cpp.png
this example looks awesome, what's the font inside?
![example-cpp](https://user-images.githubusercontent.com/8032840/192083154-fdacd4d7-783d-458f-b77a-12ca42787b44.png)

**Comment by ImADrafter** [comment]
The font is https://github.com/IBM/plex for sure. The hashtags are pretty special.

I would use the patched one from NerdFonts, its called BlexMono
